examples/radosgw-admin-example.py

```diff
@@ -77,16 +77,20 @@
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-H', '--hostname', help='radosgw hostname', required=True)
+    parser.add_argument('-p', '--port', help='radosgw port', required=True)
```

Comment:
Why is the port option required? By default it will use 443.

```suggestion
    parser.add_argument('-p', '--port', help='radosgw port', default=443)
```